src/__tests__/issue.test.ts

```diff
@@ -0,0 +1,246 @@
+import { Issue } from '../issue';
+
+test('removes excluded characters from title, when initialized', () => {
+  process.env.INPUT_PARAMETERS = JSON.stringify([{}]);
+  const content = ['(aws-cognito): This is a title'];
+
+  const issue = new Issue(content);
+
+  expect(issue['titleIssueWords']).toStrictEqual(["", "aws-cognito", ":", "This", "is", "a", "title"]);
+});
+
+test('removes excluded characters from body, when initialized', () => {
+  process.env.INPUT_PARAMETERS = JSON.stringify([{}]);
+  const content = ['(aws-cognito): This is a title', '(This) is a body'];
+
+  const issue = new Issue(content);
+
+  expect(issue['bodyIssueWords']).toStrictEqual(["", "This", "", "is", "a", "body"]);
```

Comment:
This and the test above call out a bug I hadn't realized I introduced. The empty strings being counted as words is going to result in different scoring outcomes for the title, since each word is valued differently. I'll create a bug report for this